x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/job_creator.ts

```diff
@@ -57,26 +59,54 @@ export class JobCreator {
     stop: boolean;
   } = { stop: false };
 
+  private _unsubscribeAll = new Subject();
+  private modelMemoryEstimator: ModelMemoryEstimator;
```

Comment:
for consistency all new private and protected variables and methods should start with an underscore

Reply:
this has been moved away from the job creator eventually